eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -59,8 +60,12 @@ function Should-Process-Package($pkgPath, $packageName)
     }
     # Get package info from json file created before updating version to daily dev
     $pkgInfo = Get-Content $pkgPropPath | ConvertFrom-Json
-    Write-Host "SDK Type: $($pkgInfo.SdkType)"
-    return $pkgInfo.IsNewSdk
+    $packagePath = $pkgInfo.DirectoryPath
+    $modifiedFiles = git diff --name-only --relative $TargetBranch HEAD
+    $modifiedFiles = $modifiedFiles.Where({$_.startswith($packagePath)})
```

Comment:
@sima-zhu it might make sense to make your common git diff changes also be an importable powershell function via our common helpers, rather than just a pipeline template.